lib/executor/helpers.go

```diff
@@ -33,6 +38,32 @@ func getStagesUnscaledMaxTarget(unscaledStartValue int64, stages []Stage) int64
 	return max
 }
 
+// Validates all the VU up and down shifts. For any shift that is larger than maxVUShift it will append an error.
```

Comment:
As mentioned in another comment, we're not quite checking the VUs shift, but also iteration rate in other contexts. Once we have established a better name, we should probably rephrase this comment to reflect the change too 🙇‍♂️